value_test.go

```diff
@@ -244,7 +244,7 @@ func (s *ValueS) TestGetRedirectWithoutLocation(c *C) {
 	testServer.PrepareResponse(303, headers, `{"ok": true}`)
 	v := lpad.NewValue(nil, "", testServer.URL+"/myvalue", nil)
 	_, err := v.Get(nil)
-	c.Assert(err, ErrorMatches, "Get : 303 response missing Location header")
+	c.Assert(err, ErrorMatches, "Server returned %d without Location")
```

Comment:
Making the mistake once is normal. Testing the mistake, though... :P

Reply:
Have you read the error message? :)

Reply:
Oops, the linter was even warning me... Sorry about that!

Anyway, I'm going for a more surgical approach, wait for the next commit :)